db/migrate/20160910043424_add_remaining_cached_votes_fields_and_indexes.rb

```diff
@@ -0,0 +1,29 @@
+class AddRemainingCachedVotesFieldsAndIndexes < ActiveRecord::Migration
+  def up
+    add_column :cards, :cached_votes_score, :integer, default: 0
+    add_column :cards, :cached_votes_up, :integer, default: 0
+    add_column :cards, :cached_votes_down, :integer, default: 0
+    add_column :cards, :cached_weighted_score, :integer, default: 0
```

Comment:
Hey @jimmynguyc, Since we are not using the weights on votes, do you think its better if we remove the cached_weighted_* columns? same with cached_votes_score.